### DISCUSSION TITLE
The genetic nomenclature of recessive cerebellar ataxias

### DISCUSSION THREAD

**Discussion by pnrobinson** [monarch-initiative/mondo#4031] [Bigger project]
Open question:

Is this still something we want to be done?

_Updated 10/21/21 by NV_

----

It seems this article would be a good basis for working on the ataxia part of Mondo
https://onlinelibrary.wiley.com/doi/abs/10.1002/mds.27415

**Comment by nicolevasilevsky** [comment]
Hi @pnrobinson is this still something you'd like to be done? 

Note to self - I started a spreadsheet [here](https://docs.google.com/spreadsheets/d/1tNLV3XVqkZJ3yFe6KJi8Uyb1_0_ZOrywepJOZDqOulc/edit#gid=0) with a list of the new names and OMIM xrefs

**Comment by nicolevasilevsky** [comment]
[mds.27415.docx](https://github.com/monarch-initiative/mondo/files/7368598/mds.27415.docx)
Here is the paper in Word format. I'd need to figure out a faster way to extract the lists from the table.
